### ISSUE TITLE
`SokuLib::kernel32.GetPrivateProfileIntA` wrong function table

### ISSUE THREAD

**Issue by jyanf** [SokuDev/ModTemplate#3]
Hooking `SokuLib::kernel32.GetPrivateProfileIntA` (defined in `KERNEL32.DLL.hpp`, address 0x857128) failed.
It actually leads to the function that reads IP from clipcorad in vanilla "VS Network"
The true address is 0x85712C.

**Comment by PinkySmile** [comment]
Good catch, though the right place to report this is https://github.com/SokuDev/SokuLib. Has been fixed on the latest commit of the dev branch in SokuLib.
